netgames/dmfc/dmfccfg.cpp

```diff
@@ -48,6 +48,8 @@
  * $NoKeywords: $
  */
 
+#include <string.h>
```

Comment:
Is this the import we need? This is a C import...

Reply:
It was a leftover, removed